src/dmd/mars.d

```diff
@@ -645,7 +644,7 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
         if (params.moduleDepsFile)
         {
             auto deps = File(params.moduleDepsFile);
-            deps.setbuffer(cast(void*)ob.data, ob.offset);
+            deps.setbuffer(ob.data, ob.offset);
```

Comment:
Breaking change; `params.moduleDeps` keeps on owning the buffer. Previously, ownership was transferred to `deps`, but `params.moduleDeps` was NOT reset.